### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by rebaz36**
[Portfolio: mobile menu](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md)

- create mobile menu function using javascript
- modify html accordingly
- stylize with css

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -6,10 +12,39 @@ header {
...
+
+.mobile-nav-content {
+  color: #fff;
+  padding: 20px;
+  width: 95%;
   margin: 0;
+  text-align: left;
+  font-size: 40px;
+  font-weight: 400;
+}
```
Comment (Reem-lab) [inline css/style.css]: - [x] Great job but I think that If I scroll i will see all the sections but depend on the Figma template I should just see the menu without scrolling in the side so kindly fix it to match the Figma.🔥🔥💪🏻💪🏻✅

File: index.html [hunk 2]
```diff
@@ -2,30 +2,49 @@
...
+
+            <div class="mobile-nav" id="mobile-nav">
+                <ul class="mobile-nav-content">
+                    <li class="close fa-solid fa-x fa-xs"></li>
+                    <li class="nav-item" ><a herf="#Portfolio">Portfolio</a></li>
+                    <li class="nav-item" ><a herf="#About">About</a></li>
+                    <li class="nav-item" ><a herf="#Contact">Contact</a></li>
+                </ul>
+            </div>
+
```
Comment (Reem-lab) [inline index.html]: - [x] I think here you need to blur the background to match the Figma design as much as possible ✅✅

- [x] Kindly use the same font weight as the Figma template ✅💪🏻

- [x] Please also as I can see here the close button width in sont the same width in the Figma so Kindly Fix it to be the same 🔥🔥

- [x] I noticed here when I press on portfolio or about or contact it doesn't take me to the section so kindly fix to move to the pressed section ✅✅💪🏻💪🏻


- Your design
image

- Figma design
image

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi Team , 👋🏻👋🏻

Good job so far!🔥🔥🔥💪🏻💪🏻
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- No Linters error ✅✅
- Clean code ✅✅
- Great working on the Functionality ✅✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 1 above)
Comment (rebaz36) [reply css/style.css]: thanks @Reem-lab , solved this and i learnt something new while doing it. highly appreciated

File: index.html (hunk 2 above)
Comment (rebaz36) [reply index.html]: solved, thanks i appreciate the detailed and concise feedback

**Comment by rebaz36** [comment]
Thanks @Reem-lab your feedback was really helpful

image
l

**Review by Chimwemwe-127001 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @rebaz36

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors :heavy_check_mark:
- You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.
- Repo looks professional :briefcase:
- Solid pull request :scroll:
- Cheers and good luck in your next milestones! 🍀

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rebaz36** [comment]
thanks @Chimwemwe-127001

**Pull request merged**